packages/tools/src/utilities/cine/playClip.ts

```diff
@@ -0,0 +1,246 @@
+import {
+  utilities,
+  getEnabledElement,
+  StackViewport,
+} from '@cornerstonejs/core';
+import CINE_EVENTS from './events';
+import { addToolState, getToolState } from './state';
+
+const { triggerEvent } = utilities;
+
+/**
+ * Starts playing a clip or adjusts the frame rate of an already playing clip.  framesPerSecond is
+ * optional and defaults to 30 if not specified.  A negative framesPerSecond will play the clip in reverse.
+ * The element must be a stack of images
+ * @param element - HTML Element
+ * @param framesPerSecond - Number of frames per second
+ */
+function playClip(element: HTMLDivElement, framesPerSecond: number): void {
```

Comment:
It feels weird to me that `framesPerSecond` is some special option which can be added here, yet all the others need to be added to toolstate first if you want them to be set on the first play.

Can't we add all the options which aren't internal ids etc as optional params here, rather than just the framesPerSecond, i.e.:
```
playClip(element: HTMLDivElement, options: PlayClipOptions) {...}

Type PlayClipOptions =  {
  framesPerSecond?: number;
  frameTimeVector?: number[];
  reverse?: boolean;
  loop:? boolean;
  speed?: number // Also we should rename this, as its actually just for the frame time vectors. Maybe something like `frameTimeVectorSpeedMultiplier`?
```